src/main.rs

```diff
@@ -125,10 +130,12 @@ async fn main_internal() -> Result <(), String> {
         .help("Function arguments. Must be a list of `--name value` pairs or a json string with all arguments.")
         .multiple(true);
 
+    let author = "EVERLabs";
```

Comment:
EverX (see https://everx.medium.com/ton-labs-rebrands-to-everx-67a87dcb6d87)

Reply:
done